_posts/2017-03-14-pycon-pune-2017-a-wonderful-python-experience.md

```diff
@@ -0,0 +1,220 @@
+Vimmer and maintains little [vim](http://vim.org) plugins as a part of his
+interest.
+
+#### [Stephen J. Turnbull](https://twitter.com/yasegumi)
+
+Stephen professes the Dismal Science of Economics. His knowledge is deep-rooted
+just like his beard. You will enjoy discussing on Computer science, books and
+his experience of programming. He is authoring few books written in the Japanese
+language. Stephen is [Emacsite](https://www.gnu.org/s/emacs/).
+
+![stephen_turnbull]({{
+site.url}}/assets/images/stephen_turnbull.jpg)
+
+#### [Terri Oda](https://twitter.com/terriko)
+
+Terri is a security nerd. She spent most of her time exploring tools at Intel.
+Terri knows how to hide from the spying of the U.S. Government. She is leading
+[Google Summer of Code](https://summerofcode.withgoogle.com/) section from
+[Python Software Foundation](https://www.python.org/psf/). Terri is PSF
+community service award winner. If you are a student and want to take part in
+GSoC choosing Python as your organization then She is the right person to talk.
+
+#### [Florian Fuchs](https://github.com/flofuchs)
+
+His knowledge on [ReST
+API](https://en.wikipedia.org/wiki/Representational_state_transfer) construction
+is the best. He is [Falcon](https://falconframework.org/) nerd too. I enjoyed
+discussing on various authentication mechanisms for ReST API with him.  He is
+the Red hatter.
+
+![gnu_mailman_team]({{
+site.url}}/assets/images/gnu_mailman_team.jpg)
+
+#### [Nick Coghlan](https://twitter.com/ncoghlan_dev)
+
+Nick listens more than he speaks. I will advise to not disturb him if he is
+coding. He enjoys being concentrated while coding. It was the great experience
+of getting his mentorship. He is contributing to [Core
+Python](https://github.com/python/cpython) for a few decades now.  You will
+enjoy discussing on interesting code compositions with him. He is a Red Hatter.
+
+#### [Praveen Patil](https://twitter.com/_gnovi)
+
+Unfortunately, I didn't get much time to talk with Praveen during this
+conference. He is a Math teacher who teaches concepts of Mathematics using
+Python programming language. You should feel confident to speak with him on
+Python in Education and Mathematics with him.
+
+
+#### [John Hawley](https://github.com/warthog9)
+
+John is the wittiest person in my known. His lines always end with a humor. He
+hacks mostly on hardware and the GNU/Linux. Micro Python and GNU/Linux should be
+considered as his part of interests.
+
+I am sad to declare that I was unable to attend any keynote speeches because of
+the illness. Mostly I rested at the hotel or talked with people during the
+conference days.
+
+
+![pycon_pune_2017_keynote_speackers]({{
+site.url}}/assets/images/pycon_pune_2017_keynote_speackers.jpg)
+
+
+#### Volunteer Party
+
+If you are volunteering for this conference, then you will be invited to a
+volunteer dinner party. We enjoyed party colored disco lights dancing on the
```

Comment:
This sentence is saying that the lights were dancing, and they were dancing on the DJ's "bits" (ouch!). I think you meant something like "We enjoyed dancing to music from the DJ under party colored disco lights".